index.css

```diff
@@ -16,12 +16,13 @@ background-color: #1a2661;
 .overlay {
   position: fixed;
   display: flex;
+  justify-content: center;
   flex-direction: column;
   align-items: center;
   width: 100%;
   height: 100%;
   opacity: .85;
-  background-color: rgba(111, 199, 194, 0.36);
```

Comment:
Hey Lance, I think the color you're trying to change to is too blue. The lighter tone looks more modern imo. Maybe we can ask David for his opinion.